ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Please enter full details of how to reproduce this bug.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Log file**
+Please attach the log file if relevant (e.g. `output/cellfinder_2019-12-17_13-29-46.log`)
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+
+- OS: [e.g. Linux, Windows, MacOS]
+- Version [e.g. 18.04]
+- Hardware specs [e.g. 128GB RAM, Xeon 4114 CPU]
```

Comment:
Probably good to make it clear on chip architecture, like we know `arm64` for `macOS` at least is a pain in the 🍑